src/kzg10/check.rs

```diff
@@ -0,0 +1,76 @@
+use crate::Error;
```

Comment:
Maybe let's rename this file to `check_error.rs`? This would avoid any confusion with the `PC::check` algorithm.

Reply:
And maybe we can also make these methods on `KZG10` instead of making them free functions.